addons/website_sale_stock/tests/test_website_sale_stock_abandoned_cart_email.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from odoo.addons.website_sale.tests.test_website_sale_cart_abandoned import TestWebsiteSaleCartAbandoned
+
+
+class TestWebsiteSaleStockAbandonedCartEmail(TestWebsiteSaleCartAbandoned):
```

Comment:
FYI when you inherit from a test class, the tests will be ran as many times as it's been inherited, for instance your PR adds 3 tests to the test suite.